dinner/readme.md

```diff
@@ -1,21 +1,39 @@
-# Replace with recipe name
+# Chicken Nuggets with all of the sauces
 
-## Servings: 
+## Servings: 4
 
-## Time to prepare: 
+## Time to prepare: 45 minutes 
 
 ## Ingredients:
-
+Chicken (raw) <br/>
+Panko Bread Crumbs <br/>
+barbeque sauce <br/>
+honey mustard sauce <br/>
+ketchup <br/>
+regular mustard <br/>
+ranch <br/>
+sweet and sour sauce <br/>
+vegetable oil
 
 ## Equipment needed:
-
+blender <br/>
+frier <br/>
+wire drying rack
 
 ## Steps for preparation:
-
+Step 1: Put chicken in blender <br/>
+Step 2: Blend and form into desirable nugget shapes <br/>
+Step 3: Coat in bread crumbs
+Step 4: Fry in 2 inches of vegetable oil, 10 nuggets at a time<br/>
+Step 5: Drain on wire rack<br/>
+Step 6: Dip in sauces of choice<br/>
 
 
 ### Notes:
 
 
 
 ### Codemates #
+@ofeklavian </br>
+@sean-mccoy-haas </br>
+@chasethompson1992
```

Comment:
Wow is this a secret family recipe? It looks great!

Reply:
Grandma's Chicken Nuggets